src/kOS/Screen/TermWindow.cs

```diff
@@ -963,11 +961,20 @@ protected void CheckResizeDrag()
             }
         }
         
-        void ShowCharacterByAscii(char ch, int x, int y, bool reversingScreen, int charWidth, int charHeight, float brightness)
+        void DrawCursorAt(char ch, int x, int y, bool reversingScreen, int charWidth, int charHeight, float brightness)
```

Comment:
I want to say that if we printed whole lines at a time instead of individual characters we could save some time/memory.  Except our OnGui method is far cheaper than the actual code execution with the PR, so I don't really want to waste time with it.